apollo-federation/src/operation/contains.rs

```diff
@@ -11,202 +7,6 @@ use super::InlineFragmentSelection;
 use super::Selection;
 use super::SelectionSet;
 
-/// Compare two input values, with two special cases for objects: assuming no duplicate keys,
-/// and order-independence.
-///
-/// This comes from apollo-rs: https://github.com/apollographql/apollo-rs/blob/6825be88fe13cd0d67b83b0e4eb6e03c8ab2555e/crates/apollo-compiler/src/validation/selection.rs#L160-L188
-/// Hopefully we can do this more easily in the future!
-fn same_value(left: &executable::Value, right: &executable::Value) -> bool {
```

Comment:
`same_value` and `same_arguments` are removed. `==` on an ArgumentList or DirectiveList does the same thing now.